### PR TITLE
Bug 1625207 - Make device manufacturer and model optional

### PR DESCRIPTION
**Pull request by badboy**
These are less clear-cut on non-mobile.
More specific hardware identification might be necessary for Desktop use cases.

Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] If the PR comes from a fork, trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional) and review the report posted in the comments.

For glean changes:
- [x] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

**Comment by badboy** [comment]
cc @Dexterp37

File: templates/include/glean/glean.1.schema.json [hunk 1]
```diff
@@ -36,8 +36,6 @@
         "app_build",
         "app_display_version",
         "architecture",
-        "device_manufacturer",
```
Comment (Dexterp37) [inline templates/include/glean/glean.1.schema.json]: @badboy can you kindly also update the docs [here](https://mozilla.github.io/glean/book/user/pings/index.html#the-client_info-section) and the related `metrics.yaml` files once that's merged?

Comment (badboy) [reply templates/include/glean/glean.1.schema.json]: of course!

**Comment by jklukas** [comment]
Looks good. Are you ready for us to go ahead and merge?

**Comment by badboy** [comment]
> Looks good. Are you ready for us to go ahead and merge?

All fine from our side!

**Pull request merged**